cypress/e2e/update_personal_information/index.cy.ts

```diff
@@ -8,10 +8,9 @@ describe("Signup into new entreprise unipersonnelle", () => {
   });
 
   it("Should send email when user updates personal information", function () {
-    cy.login(
-      "9023e9f4-4e54-4ba0-9558-3cb61e7608c6@mailslurp.com",
-      "password123",
-    );
+    cy.visit("/personal-information");
```

Comment:
In other files, it's written with backtick like this: cy.visit(`/personal-information`);

Reply:
I just committed a change to standardize this. We can now use double quotes consistently (which is preferable when no string interpolation is needed).